src/Directory.Build.props

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <Copyright>Copyright © 2020 Your Company</Copyright>
     <Authors>Your Authors</Authors>
-    <VersionPrefix>0.3.0</VersionPrefix>
+    <VersionPrefix>0.9.0-beta3</VersionPrefix>
```

Comment:
Should set this back to 0.3.0 - understood why this was done for testing purposes, but want things to be "clean" for end-users